routing/routing_tests/applying_traffic_test.cpp

```diff
@@ -0,0 +1,198 @@
+#include "testing/testing.hpp"
+
+#include "routing/car_model.hpp"
+#include "routing/geometry.hpp"
+#include "routing/index_graph.hpp"
+#include "routing/index_graph_starter.hpp"
+
+#include "traffic/traffic_info.hpp"
+
+#include "geometry/point2d.hpp"
+
+#include "routing/base/astar_algorithm.hpp"
+
+#include "std/shared_ptr.hpp"
+#include "std/unique_ptr.hpp"
+#include "std/vector.hpp"
+
+namespace
+{
+using namespace routing;
+using namespace traffic;
+
+// @TODO(bykoianko) When PR with applying restricions is merged all the helper method
+// should be used from "routing/routing_tests/index_graph_tools.hpp".
+class TestGeometryLoader final : public routing::GeometryLoader
```

Comment:
Yes. You're right. The problem is I've done it in https://github.com/mapsme/omim/pull/4817 (not merge yet).
Probably it's worth moving routing/routing_tests/index_graph_tools.cpp form https://github.com/mapsme/omim/pull/4817 to this PR.